docs/permissions/getting-started.md

```diff
@@ -34,129 +22,44 @@ Like many other parts of Backstage, the permissions framework relies on informat
-       return { result: AuthorizeResult.ALLOW };
-     }
-   }
-
-   export default async function createPlugin(
-     env: PluginEnvironment,
-   ): Promise<Router> {
-     return await createRouter({
-       config: env.config,
-       logger: env.logger,
-       discovery: env.discovery,
-       policy: new TestPermissionPolicy(),
-       identity: env.identity,
-     });
-   }
-   ```
-
-3. Wire up the permission policy in `packages/backend/src/index.ts`. [The index in the example backend](https://github.com/backstage/backstage/blob/master/packages/backend/src/index.ts) shows how to do this. You’ll need to import the module from the previous step, create a plugin environment, and add the router to the express app:
-
-   ```ts title="packages/backend/src/index.ts"
-   import proxy from './plugins/proxy';
-   import techdocs from './plugins/techdocs';
-   import search from './plugins/search';
-   /* highlight-add-next-line */
-   import permission from './plugins/permission';
-
-   async function main() {
-     const techdocsEnv = useHotMemoize(module, () => createEnv('techdocs'));
-     const searchEnv = useHotMemoize(module, () => createEnv('search'));
-     const appEnv = useHotMemoize(module, () => createEnv('app'));
-     /* highlight-add-next-line */
-     const permissionEnv = useHotMemoize(module, () => createEnv('permission'));
-     // ..
-
-     apiRouter.use('/techdocs', await techdocs(techdocsEnv));
-     apiRouter.use('/proxy', await proxy(proxyEnv));
-     apiRouter.use('/search', await search(searchEnv));
-     /* highlight-add-next-line */
-     apiRouter.use('/permission', await permission(permissionEnv));
-     // ..
-   }
-   ```
+The permissions framework uses the `permission-backend` plugin to accept authorization requests from other plugins across your Backstage deployment. The default `@backstage/create-app` template includes the permission backend, but if you need to make the change manually, these are the steps:
+
+1. Add `@backstage/plugin-permission-backend` and `@backstage/plugin-permission-backend-module-allow-all-policy` to your backend dependencies, this will add the permission backend and a policy that allows all permissions:
+
+```bash
+# From your Backstage root directory
+yarn --cwd packages/backend add @backstage/plugin-permission-backend @backstage/plugin-permission-backend-module-allow-all-policy
+```
+
+2. Add the following to `packages/backend/src/index.ts`. This adds the permission-backend router, and configures it with a policy which allows everything.
+
+```typescript title="packages/backend/src/index.ts"
+import { createBackend } from '@backstage/backend-defaults';
+const backend = createBackend();
+// ...
+/* highlight-add-next-line */
+backend.add(import('@backstage/plugin-permission-backend/alpha'));
+/* highlight-add-next-line */
+backend.add(
+  import('@backstage/plugin-permission-backend-module-allow-all-policy'),
+);
+// ...
+backend.start();
+```
```

Comment:
do we need this? I think it makes things more complicated. We recommended to keep backstage up to date in the step before, which already includes this step. Remove it? 😄

```suggestion
```

Reply:
True, less is more as they say! 😅